### ISSUE TITLE
Use Travis builtin support for codecov and coveralls?

### ISSUE THREAD

**Issue by harryscholes** [invenia/PkgTemplates.jl#68]
Travis now adds codecov and coveralls support:

```yaml
codecov: true
coveralls: true
```

https://docs.travis-ci.com/user/languages/julia/#coverage

**Comment by christopher-dG** [comment]
IIRC that support is kind of broken in that it causes build errors instead of failures. @ararslan could you comment?

**Comment by ararslan** [comment]
Been a while since I've looked at it. I think @simonbyrne would have the most up-to-date knowledge on that, since he's the one who implemented it.

**Comment by simonbyrne** [comment]
Yes it is broken: I tried to fix it in https://github.com/travis-ci/travis-build/pull/1571, but got stuck.

**Comment by harryscholes** [comment]
Ahh okay. We could close this for now and maybe implement the changes when Travis is fixed

**Comment by christopher-dG** [comment]
I'll leave it open, since otherwise I'll probably forget about it :upside_down_face:
